src/TestCase.php

```diff
@@ -41,10 +34,6 @@ public function setUp()
         if (! $this->app) {
             $this->refreshApplication();
         }
-
-        if (! $this->factory) {
-            $this->factory = $this->app->make(Factory::class);
```

Comment:
@crynobone We're just breaking a small amount of code. Better than breaking everyone's code with fatal errors.